indico/modules/categories/client/js/calendar.js

```diff
@@ -23,6 +23,8 @@ import CalendarLegend from './components/CalendarLegend';
 (function(global) {
   let groupBy = 'category';
   const filteredLegendElements = new Set();
+  const filteredKeywords = new Set();
+  const filteringByKeyword = () => groupBy === 'keywords';
```

Comment:
If one day the same logic has to be applied somewhere else people might tend to duplicate code instead of using the function. I honestly prefer my approach, but if you insist I'll change it.

Reply:
well it's an extra function call that happens for each event but no, i don't care strongly enough ;)